segwit/README.md

```diff
@@ -33,3 +33,4 @@
 1. How did users know whether miners support SegWit prior to activation?
 1. How does a (virtual block increase) affect IBD cost over time?
 1. Where are the locking script operations in P2WPKH/P2WSH?
+1. What are some address format design goals and requirements discussed in the bech32 talk? How problematic is the [bech32 mutability](https://bitcoinops.org/en/newsletters/2019/12/28/#bech32-mutability) issue for v0 segwit addresses versus for taproot?
```

Comment:
1. This refers to an optional resource, so not sure how to indicate this question is only for those that watched the video.

2. I think the optech link should go in the answers file otherwise it just becomes another thing to read beforehand.

Reply:
Could put [optional] on it? and move the optech link to answers.md, yeah